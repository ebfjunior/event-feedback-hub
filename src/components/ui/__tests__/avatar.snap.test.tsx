import React from 'react';
import { render } from '@testing-library/react';
import { Avatar, AvatarFallback, AvatarImage } from '../avatar';

describe('ui/Avatar (snapshot)', () => {
	it('renders image and fallback', () => {
		const { container } = render(
			<Avatar>
				<AvatarImage src="/avatar.png" alt="A" />
				<AvatarFallback>AB</AvatarFallback>
			</Avatar>,
		);
		expect(container.firstChild).toMatchSnapshot();
	});
});


