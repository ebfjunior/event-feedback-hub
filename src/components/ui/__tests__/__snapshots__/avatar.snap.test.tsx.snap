// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ui/Avatar (snapshot) > renders image and fallback 1`] = `
<span
  class="relative flex size-8 shrink-0 overflow-hidden rounded-full"
  data-slot="avatar"
>
  <span
    class="bg-muted flex size-full items-center justify-center rounded-full"
    data-slot="avatar-fallback"
  >
    AB
  </span>
</span>
`;
